conda_smithy/templates/travis.yml.tmpl

```diff
@@ -4,7 +4,8 @@
 language: generic
 
 os: osx
-osx_image: xcode6.4
+osx_image: {{ travis.get('osx_image', 'xcode9.4') }}
```

Comment:
Should we just fix it to `xcode9.4`? We've been working on building on Azure. So having this be configurable may not be necessary. Besides if we continue to use Travis CI, we have a few years before we need to think about this again.

Reply:
I'm +1 to making it configurable. There's no harm in it.